cirrus/server/cirrus/main.py

```diff
@@ -137,10 +171,17 @@ async def compute_features(request_data: FeatureRequest):
     enrolled_partial_configuration: Dict[str, Any] = app.state.sdk.compute_enrollments(
         targeting_context
     )
+    print(enrolled_partial_configuration)
```

Comment:
I think we can remove this now that it's working?

Reply:
OOP I thought I pulled all those out 😅